### PR TITLE
CCDB path as argument to reconstruction

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: Detectors/FIT/FT0/workflow/include/FT0Workflow/ReconstructionSpec.h [hunk 1]
```diff
@@ -34,14 +34,15 @@ class ReconstructionDPL : public Task
   static constexpr int NCHANNELS = o2::ft0::Geometry::Nchannels;
 
  public:
-  ReconstructionDPL(bool useMC) : mUseMC(useMC) {}
+  ReconstructionDPL(bool useMC, std::string ccdbpath) : mUseMC(useMC), mCCDBpath(ccdbpath) {}
```
Comment (shahor02) [inline Detectors/FIT/FT0/workflow/include/FT0Workflow/ReconstructionSpec.h:37]: could you change all such arguments to `const std::string& ccdbpath`?

File: Detectors/FIT/macros/uploadLookUpTable.C [hunk 2]
```diff
@@ -37,28 +37,7 @@ void uploadLookUpTable()
   }
   CcdbApi api;
   std::map<std::string, std::string> metadata; // can be empty
-  api.init("http://ccdb-test.cern.ch:8080/");  // or http://localhost:8080 for a local installation
+  api.init(url);
   // store abitrary user object in strongly typed manner
   api.storeAsTFileAny(&table, "FT0/LookUpTable", metadata);
 }
-/*
```
Comment (shahor02) [inline Detectors/FIT/macros/uploadLookUpTable.C:44]: could this code be removed, if obsolete?

**Review by shahor02 — CHANGES_REQUESTED** [review]

**Comment by AllaMaevskaya** [comment]
I did all what you requested

**Comment by shahor02** [comment]
Hi @AllaMaevskaya 

The FullCI fails with error:
````
[16926:ft0-reconstructor]: [ERROR] error parsing options of o2-ft0-reco-workflow: option '--ccdb-path' is ambiguous and matches different versions of '--ccdb-path'
````
Could you rename the option to `--ccdb-path-ft0` ?

**Comment by AllaMaevskaya** [comment]
build/O2/fullCI error is unrelated

**Comment by AllaMaevskaya** [comment]
Is it OK now?

**Review by shahor02 — APPROVED** [review]

**Pull request merged**